drop_cap/demo/src/main/res/layout/activity_drop_cap.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  android:orientation="vertical">
+
+  <FrameLayout
```

Comment:
this is just a divider? Is FrameLayout the lightest thing you can use? maybe `Space` or `View`